compiler/plugins/target/ROCM/ROCMTargetUtils.cpp

```diff
@@ -39,6 +40,18 @@ bool couldNeedDeviceBitcode(const llvm::Module &module) {
   return false;
 }
 
+bool couldNeedUkernelBitcode(const llvm::Module &module) {
```

Comment:
Do we really need this? Looking for specific function names is a bit strange for me (maybe I did this earlier too, if so apologies). The linker itself should make this unnecessary.

Reply:
XLA-ROCM backend was handling bitcode linking this way haha